jid/escape_test.go

```diff
@@ -44,7 +45,7 @@ var unescapeTestCases = [...]struct {
 	5: {`nothingtodohere`, `nothingtodohere`, false, 15, nil, nil},
 	6: {`a\a\20`, `a\a `, false, 3, nil, transform.ErrEndOfSpan},
 	7: {`aa\2`, `aa\2`, true, 4, nil, nil},
-	8: {`aa\2`, `aa`, false, 2, transform.ErrShortSrc, transform.ErrShortSrc},
```

Comment:
Why were the tests changed? I believe the test that was removed would be failing (indicating a problem) with the current PR and the two new tests don't actually add any test coverage. Do they test something specific that I'm missing?